templates/lightbox.html

```diff
@@ -0,0 +1,36 @@
+<style>
+    .lightbox {
+        position: fixed;
+        z-index: 1000;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        background-color: rgba(22,27,34,0.8);
+        display: none;
+    }
+
+    .lightbox-active {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+</style>
+<script>
```

Comment:
Oh god... are we really supposed to use `<style>`? surely a CSS file is preferable.

Reply:
I feel like there isn't many style code so this is fine?